### PR TITLE
Fix #840: See more added to TopicInfo

### LINKED ISSUE #840: Lowfi: TopicInfo needs `See More button`
TopicInfo needs see more button.
Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/19cfbacf-854c-4c7d-8691-3b3d117e1866/TP-Overview-/

Description can be of max character from 100  → 240 characters
Therefore `See More` button has been introduced.
The textview should be: 5 lines long when collapsed and 12 lines max when expanded

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes: #840 

Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/19cfbacf-854c-4c7d-8691-3b3d117e1866/TP-Overview-/

I have actually made some more changes to this to be on the safer side, if the description takes less than 6 lines, then see more will not be visible else it will be visible.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt [hunk 1]
```diff
@@ -93,9 +99,62 @@ class TopicInfoFragmentTest {
...
+  @Test
+  fun testTopicInfoFragment_loadFragment_clickSeeMore_textChangesToSeeLess() {
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      onView(withId(R.id.see_more_text_view)).perform(click())
+      onView(withId(R.id.see_more_text_view)).check(matches(withText(R.string.see_less)))
+    }
+  }
+
+  @Test
+  fun testTopicInfoFragment_loadFragment_clickSeeMore_topicDescriptionLinesChangeToSix() {
```
Comment (veena14cs) [inline app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt]: This testcase is failing.
Screenshot 2020-03-30 at 11 42 23 AM

File: app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt [hunk 2]
```diff
@@ -93,9 +99,62 @@ class TopicInfoFragmentTest {
...
 
+  @Test
+  fun testTopicInfoFragment_loadFragment_checkDefaultTopicDescriptionLines_fiveLinesVisible() {
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      onView(withId(R.id.topic_description_text_view)).check(matches(maxLines(/* lineCount= */ 5)))
+    }
+  }
+
+  @Test
+  fun testTopicInfoFragment_loadFragment_seeMoreIsVisible() {
```
Comment (veena14cs) [inline app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt:111]: This testcase is failing.
Screenshot 2020-03-30 at 11 49 51 AM

**Review by veena14cs — CHANGES_REQUESTED** [review]
Some testcases are failing please take a look.

**Comment by rt4914** [comment]
> Some testcases are failing please take a look.

All the test cases were passing correctly on my device. So maybe this might be the case if your device is in landscape mode. Anyways, I have added `scrollTo` step to all test cases. PTAL

**Review by veena14cs — CHANGES_REQUESTED** [review]
Still it is failing. I checked In portrait mode.
Screenshot 2020-03-30 at 1 18 08 PM

**Review by veena14cs — APPROVED** [review]
These testcase are still failing in Pixel 3A XL real device `testTopicInfoFragment_loadFragment_clickSeeMore_textChangesToSeeLess` and 
`testTopicInfoFragment_loadFragment_seeMoreIsVisible`. 
I checked in emulator Pixel XL its passing so I am approving it.

**Comment by rt4914** [comment]
> These testcase are still failing in Pixel 3A XL real device `testTopicInfoFragment_loadFragment_clickSeeMore_textChangesToSeeLess` and
> `testTopicInfoFragment_loadFragment_seeMoreIsVisible`.
> I checked in emulator Pixel XL its passing so I am approving it.

Filed an issue #914 to track this.

**Pull request merged**